src/Expr.cc

```diff
@@ -3218,12 +3218,16 @@ void HasFieldExpr::ExprDescribe(ODesc* d) const
 		d->Add(field);
 	}
 
+
 RecordConstructorExpr::RecordConstructorExpr(ListExprPtr constructor_list)
-	: Expr(EXPR_RECORD_CONSTRUCTOR), op(std::move(constructor_list))
```

Comment:
What's the reason for moving this? It should initialize the same way. The initialization of `map` could be added here as well.

Reply:
I don't recall why I did, probably vestigial from how the code looked a long while ago when I first added that (basically a year ago!).  Reverted.